### ISSUE TITLE
Handling uncaught errors in the iodide notebook

### ISSUE THREAD

**Issue by rth** [iodide-project/pyodide#173]
Currently some errors are just shown in the console log, which can be confusing for users.

For instance below are some errors I recently got, while trying to load `iodide.js` (just to illustrate my point, the actual error message is not relevant),

> uncaught exception: out of memory 

and

> Uncaught (in promise) TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.

both of these are shown as errors in the console all the while the iodide cell was saying,

> python plugin downloaded, initializing

Another example, is that providing a non existing URL for `pyodide.js` will not display any errors in iodide.

This is confusing for users, who might not necessarily know (or want) to look at the console. Maybe it's an issue more for iodide though..

**Comment by mdboom** [comment]
Yeah, I would definitely say this is an iodide issue.  Would you mind reporting this there?
